File: android/app/src/main/java/com/cec/mini/calory/models/CaloriesDaily.java
package com.cec.mini.calory.models;

public class CaloriesDaily {

    private String foodName;

    private double weight;

    private double calories;

    private String date;

    private String emailId;
    private String meal;

    private String image;

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getMeal() {
        return meal;
    }

    public void setMeal(String meal) {
        this.meal = meal;
    }

    public String getEmailId() {
        return emailId;
    }

    public void setEmailId(String emailId) {
        this.emailId = emailId;
    }

    public String getFoodName() {
        return foodName;
    }

    public void setFoodName(String foodName) {
        this.foodName = foodName;
    }

    public double getWeight() {
        return weight;
    }

    public void setWeight(double weight) {
        this.weight = weight;
    }

    public double getCalories() {
        return calories;
    }

    public void setCalories(double calories) {
        this.calories = calories;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public double getProtein() {
        return protein;
    }

    public void setProtein(double protein) {
        this.protein = protein;
    }

    public double getCarbohydrate() {
        return carbohydrate;
    }

    public void setCarbohydrate(double carbohydrate) {
        this.carbohydrate = carbohydrate;
    }

    public double getFat() {
        return fat;
    }

    public void setFat(double fat) {
        this.fat = fat;
    }

    public boolean isConsumed() {
        return consumed;
    }

    public void setConsumed(boolean consumed) {
        this.consumed = consumed;
    }

    private double protein;

    private double carbohydrate;

    private double fat;

    private boolean consumed;



}

File: lib/src/common/models/calories_daily.dart
class CaloriesDaily {
  final String foodName;
  final double weight;
  final double calories;
  final String date;
  final String emailId;
  final String meal;
  final String image;
  final double protein;
  final double carbohydrate;
  final double fat;
  final bool consumed;

  CaloriesDaily({
    required this.foodName,
    required this.weight,
    required this.calories,
    required this.date,
    required this.emailId,
    required this.meal,
    required this.image,
    required this.protein,
    required this.carbohydrate,
    required this.fat,
    required this.consumed,
  });

  factory CaloriesDaily.fromJson(Map<String, dynamic> json) {
    return CaloriesDaily(
      foodName: json['foodName'],
      weight: json['weight'],
      calories: json['calories'],
      date: json['date'],
      emailId: json['emailId'],
      meal: json['meal'],
      image: json['image'],
      protein: json['protein'],
      carbohydrate: json['carbohydrate'],
      fat: json['fat'],
      consumed: json['consumed'],
    );
  }
}
